git-event/handler.go

```diff
@@ -27,7 +29,6 @@ func Handle(req []byte) string {
 			}
 		}
 
-		event := InstallationRepositoriesEvent{}
```

Comment:
After auditEvent is moved to the if block, this can stay here. But this is not an issue. Feel free to ignore the comment if you don't have time to address.